modin/engines/ray/pandas_on_ray/frame/partition.py

```diff
@@ -17,11 +17,9 @@
 
 from modin.data_management.utils import length_fn_pandas, width_fn_pandas
 from modin.engines.base.frame.partition import PandasFramePartition
-from modin.engines.ray.utils import handle_ray_task_error
```

Comment:
why we no longer need that?

Reply:
Ray throws the correct error Type/Message now instead of wrapping in their own.

Reply:
If you look at how we were handling this before, you would cringe. It was a filthy hack.

https://github.com/modin-project/modin/pull/3132/files#diff-72036370d85971ebd29660bf18d7dc4eb75e56609955d5291b23750e2bf1c28aL34

Reply:
I remember that function (even without looking it up), ugh!, and I'm glad it's now gone 😄 was just double-checking.